### PR TITLE
[docs] [drivers and ORMs] Drivers and ORMs for stable version.

### LINKED ISSUE #13365: [Docs] Make changes to Drivers and ORMs in stable.
### Description

- Modify links, description and any relevant changes that's in preview to stable.

### PR DESCRIPTION
**Pull request by aishwarya24**
resolves #13365 

@netlify /stable/drivers-orms/

### CODE REVIEW

File: docs/content/stable/drivers-orms/java/yugabyte-jdbc.md [hunk 1]
```diff
@@ -16,14 +16,14 @@ For Java applications, the JDBC driver provides database connectivity through th
...
       YugabyteDB JDBC Driver
     </a>
   </li>
 
   <li >
-    <a href="/preview/drivers-orms/java/postgres-jdbc/" class="nav-link">
+    <a href="../postgres-jdbc/" class="nav-link">
       <i class="icon-postgres" aria-hidden="true"></i>
       PostgreSQL JDBC Driver
     </a>
```
Comment (ddhodge) [inline docs/content/stable/drivers-orms/java/yugabyte-jdbc.md:29]: fix YSQL link

File: docs/content/stable/drivers-orms/nodejs/postgres-node-driver.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ type: docs
 <ul class="nav nav-tabs-alt nav-tabs-yb">
 
   <li >
-    <a href="/preview/drivers-orms/nodejs/postgres-node-driver/" class="nav-link active">
+    <a href="../drivers-orms/nodejs/postgres-node-driver/" class="nav-link active">
```
Comment (ddhodge) [inline docs/content/stable/drivers-orms/nodejs/postgres-node-driver.md:16]: broken link

File: docs/content/stable/drivers-orms/nodejs/postgres-node-driver.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ type: docs
...
       <i class="icon-postgres" aria-hidden="true"></i>
       PostgreSQL node-postgres Driver
     </a>
```
Comment (ddhodge) [inline docs/content/stable/drivers-orms/nodejs/postgres-node-driver.md:19]: fix build an app link

File: docs/content/stable/drivers-orms/nodejs/prisma.md [hunk 4]
```diff
@@ -14,13 +14,13 @@ type: docs
...
       <i class="fab fa-node-js" aria-hidden="true"></i>
       Sequelize
     </a>
   </li>
   <li>
-    <a href="/preview/drivers-orms/nodejs/prisma/" class="nav-link active">
+    <a href="../prisma/" class="nav-link active">
       <i class="fab fa-node-js" aria-hidden="true"></i>
       Prisma
     </a>
```
Comment (ddhodge) [inline docs/content/stable/drivers-orms/nodejs/prisma.md:26]: fix scaling link at bottom

File: docs/content/stable/drivers-orms/nodejs/sequelize.md [hunk 5]
```diff
@@ -20,7 +20,7 @@ type: docs
     </a>
   </li>
   <li >
-    <a href="/preview/drivers-orms/nodejs/prisma/" class="nav-link ">
+    <a href="../prisma/" class="nav-link ">
       <i class="fab fa-node-js" aria-hidden="true"></i>
       Prisma
     </a>
```
Comment (ddhodge) [inline docs/content/stable/drivers-orms/nodejs/sequelize.md:26]: fix scaling link at bottom

File: docs/content/stable/drivers-orms/python/django.md [hunk 6]
```diff
@@ -13,14 +13,14 @@ type: docs
...
       Django ORM
     </a>
   </li>
 
   <li >
-    <a href="/preview/drivers-orms/python/sqlalchemy/" class="nav-link">
+    <a href="../sqlalchemy/" class="nav-link">
       <i class="icon-postgres" aria-hidden="true"></i>
       SQLAlchemy ORM
     </a>
```
Comment (ddhodge) [inline docs/content/stable/drivers-orms/python/django.md:26]: fix build an app link
YB Managed link at bottom broken - just remove it

File: docs/content/stable/drivers-orms/python/postgres-psycopg2.md [hunk 7]
```diff
@@ -14,14 +14,14 @@ type: docs
...
       YugabyteDB Psycopg2
     </a>
   </li>
 
   <li >
-    <a href="/preview/drivers-orms/python/postgres-psycopg2" class="nav-link active">
+    <a href="../postgres-psycopg2" class="nav-link active">
       <i class="icon-postgres" aria-hidden="true"></i>
       PostgreSQL Psycopg2
     </a>
```
Comment (ddhodge) [inline docs/content/stable/drivers-orms/python/postgres-psycopg2.md:27]: fix build an app link

File: docs/content/stable/drivers-orms/python/sqlalchemy.md [hunk 8]
```diff
@@ -14,14 +14,14 @@ type: docs
...
       Django ORM
     </a>
   </li>
 
   <li >
-    <a href="/preview/drivers-orms/python/sqlalchemy/" class="nav-link active">
+    <a href="../sqlalchemy/" class="nav-link active">
       <i class="icon-postgres" aria-hidden="true"></i>
       SQLAlchemy ORM
     </a>
```
Comment (ddhodge) [inline docs/content/stable/drivers-orms/python/sqlalchemy.md:27]: fix build and scaling links

File: docs/content/stable/drivers-orms/python/yugabyte-psycopg2.md [hunk 9]
```diff
@@ -14,14 +14,14 @@ type: docs
...
       YugabyteDB Psycopg2
     </a>
   </li>
 
   <li >
-    <a href="/preview/drivers-orms/python/postgres-psycopg2" class="nav-link">
+    <a href="../postgres-psycopg2" class="nav-link">
       <i class="icon-postgres" aria-hidden="true"></i>
       PostgreSQL Psycopg2
     </a>
```
Comment (ddhodge) [inline docs/content/stable/drivers-orms/python/yugabyte-psycopg2.md:27]: fix Build link
YBM link broken

**Review by ddhodge — COMMENTED** [review]
misc comments

File: docs/content/stable/drivers-orms/csharp/entityframework.md [hunk 10]
```diff
@@ -14,7 +14,7 @@ type: docs
 
 <ul class="nav nav-tabs-alt nav-tabs-yb">
   <li >
-    <a href="/preview/drivers-orms/csharp/entityframework/" class="nav-link active">
+    <a href="../entityframework/" class="nav-link active">
       <i class="icon-postgres" aria-hidden="true"></i>
       EntityFramework ORM
     </a>
```
[the remark this answers is not in the document]
Comment (aishwarya24) [reply docs/content/stable/drivers-orms/csharp/entityframework.md:20]: removing the YBM links for stable on all language pages.

File: docs/content/stable/drivers-orms/go/_index.md [hunk 11]
```diff
@@ -12,43 +12,50 @@ menu:
 type: indexpage
```
[the remark this answers is not in the document]
Comment (aishwarya24) [reply docs/content/stable/drivers-orms/go/_index.md:12]: Sorry, copy-paste accidentally may have resulted in this issue. Will fix it.

File: docs/content/stable/drivers-orms/go/_index.md [hunk 12]
```diff
@@ -45,10 +45,10 @@ To develop Golang applications for YugabyteDB, you need the following:
   For ease-of-use, use an integrated development environment (IDE) such as Visual Studio. To download and install Visual Studio, visit the [Visual Studio Downloads](https://visualstudio.microsoft.com/downloads/) page.
 
 - **YugabyteDB cluster**
-  - Create a free cluster on [YugabyteDB Managed](https://www.yugabyte.com/cloud/). Refer to [Create a free cluster](../../yugabyte-cloud/cloud-basics/create-clusters-free/). Note that YugabyteDB Managed requires SSL.
-  - Alternatively, set up a standalone YugabyteDB cluster by following the steps in [Install YugabyteDB](/preview/quick-start/install/macos).
+  - Create a free cluster on [YugabyteDB Managed](https://www.yugabyte.com/cloud/). Refer to [Use a cloud cluster](../../quick-start-yugabytedb-managed/). Note that YugabyteDB Managed requires SSL.
```
Comment (ddhodge) [inline docs/content/stable/drivers-orms/go/_index.md:48]: broken link

**Review by ddhodge — APPROVED** [review]
LGTM

**Pull request merged**